test/extended/util/client.go

```diff
@@ -931,7 +932,15 @@ func (c *CLI) start(stdOutBuff, stdErrBuff *bytes.Buffer) (*exec.Cmd, error) {
 	}
 	cmd := exec.Command(c.execPath, c.finalArgs...)
 	cmd.Stdin = c.stdin
-	framework.Logf("Running '%s %s'", c.execPath, strings.Join(c.finalArgs, " "))
+	// The below log line prints the bearer token in the logs hence redacting
+	// the log if it contains bearer token
+	args := strings.Join(c.finalArgs, " ")
+	if strings.Contains(args, "Bearer") {
+		// replace bearer token with XXXXX string
+		re := regexp.MustCompile(`Authorization:\s+Bearer.*\s+`)
+		args = re.ReplaceAllString(args, "XXXXXXXXXXXXXX ")
+	}
```

Comment:
Instead of adding this directly to the `start` function, please create a small function, e.g. `redactBearerToken` and call it in the log line like this:
```
framework.Logf("Running '%s %s'", c.execPath, redactBearerToken(args))
```

Reply:
Addressed